protocol/src/main.ts

```diff
@@ -157,7 +157,7 @@ export interface ProtocolConnection {
 	/**
 	 * Enables tracing mode for the connection.
 	 */
-	trace(value: Trace, tracer: Tracer, sendNotification?: boolean): void;
+	trace(value: Trace, traceFormat: TraceFormat, tracer: Tracer, sendNotification?: boolean): void;
```

Comment:
This is breaking as well. We should support this in a non breaking way by having two signatures. If the old one is used we do the old text tracing.